docs/install/rpm.md

```diff
@@ -40,6 +40,8 @@ Installing and running Open Distro for Elasticsearch from an RPM package is a mo
 
 1. Choose the version you'd like and install it:
 
+   The below command will pick the appropriate ARCH(x64 or ARM64) based on the system being used. 
```

Comment:
Maybe replace "ARCH" with architecture just to be clear. Avoid acronyms

Reply:
made the change.